### DISCUSSION TITLE
Moving the X & Y with machine turned on???

### DISCUSSION THREAD

**Discussion by ghost** [arkypita/LaserGRBL#2065] [General]
I watched a You Tube Video where the person was able to move his X and Y with the machine turned on is this normal, I have a Sainsmart LC-40 and when it is turned on I cannot move the axis I must turn the machine off

**Comment by ivdor** [comment]
You can disable motors on gcode machines by sending M18 command. Or read the f... manual ))
I also added M18 to the "focus" button, for manual setting of 0,0 coordinates
